junit-platform-engine/src/main/java/org/junit/platform/engine/ConfigurationParameters.java

```diff
@@ -65,6 +65,13 @@
 	 */
 	Optional<String> get(String key);
 
+	/**
+	 * @since 1.5
+	 */
+	default String get(ConfigurationParameter configurationParameter) {
```

Comment:
The default overload feels logically correct, but leads to ambigous resolution errors. See new casts below. Perhaps a signature like `String getOrDefault(ConfigurationParameter)` would be better.